### PR TITLE
updated Makefiles and documentation to work with large data sets

### PR DESCRIPTION
**Pull request by vpbrendel**
fasterq-dump seems to be the way to go, with appropriate options in case of large data sets

### CODE REVIEW

File: HOWTO.md [hunk 1]
```diff
@@ -213,12 +213,29 @@ Take a look and explore.  The _.stats_ and _report_ files would be good
...
+For large data sets, the simple _fasterq\_dump_ command in the _Makefile_
+may not be the best choice.
+You may want to review the options to _fasterq\_dump_.
+For example,
+
+```
+fasterq-dump SRRaccesion -e 8 -t /dev/shm -p
+```
+
+would use 8 processors to download _SRRaccessioni_ and put the result into
```
Comment (katemortensen) [inline HOWTO.md]: _SRRaccession_ rather?

**Review by katemortensen — COMMENTED** [review]
The only thing I noticed (very minor) was on line 226 of the HOWTO.md page, did you mean SRRaccession rather than SRRaccessioni?

File: HOWTO.md (hunk 1 above)
Comment (vpbrendel) [reply HOWTO.md]: Good find. Fixed (see added commits below).

**Review by katemortensen — APPROVED** [review]
Looks good!

**Pull request merged**